### PR TITLE
Added EN translation of developer instructions

### LINKED ISSUE #7569: Translate developer instruction.
https://github.com/yandex/ClickHouse/blob/master/dbms/tests/instructions/developer_instruction_ru.md

### PR DESCRIPTION
**Pull request by Akazz**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation

Changelog entry (up to few sentences, required except for Non-significant/Documentation categories):
Added EN translation of developer instructions in the `dbms/tests/instructions` section.

Closes #7569

### CODE REVIEW

File: dbms/tests/instructions/developer_instruction_en.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
+Building of ClickHouse is supported on Linux, FreeBSD and Mac OS X.
+
+# If you use Windows
+
+If you use Windows, you need to create a virtual machine with Ubuntu. To start working with a virtual machine please install VirtualBox. You can download Ubuntu from the website: https://www.ubuntu.com/#download Please create a virtual machine from the downloaded image (you should reserve at least 4GB of RAM for it). To run a command line terminal in Ubuntu, please locate a program containing the word "terminal" in its name (gnome-terminal, konsole etc.) or just press Ctrl+Alt+T.
+
+
+# Creating a repository on GitHub
+
+To start working with ClickHouse repository you will need a GitHub account.
```
Comment (GoToCoding) [inline dbms/tests/instructions/developer_instruction_en.md]: `you will need` -> `you need`

File: dbms/tests/instructions/developer_instruction_en.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+Probably, you already have one. If you don't, please register at https://github.com In case you do not have ssh keys, you should generate them and then upload them on GitHub. It is required for sending over your patches. It is also possible to use the same ssh keys that you use with any other ssh servers - probably you already have those.
```
Comment (GoToCoding) [inline dbms/tests/instructions/developer_instruction_en.md]: dot after `at https://github.com`

File: dbms/tests/instructions/developer_instruction_en.md [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+
+Create a fork of ClickHouse repository. To do that please click on the "fork" button in the upper right corner at https://github.com/ClickHouse/ClickHouse  It will fork your own copy of ClickHouse/ClickHouse to your account.
```
Comment (GoToCoding) [inline dbms/tests/instructions/developer_instruction_en.md]: dot after `ClickHouse/ClickHouse` ?

File: dbms/tests/instructions/developer_instruction_en.md [hunk 4]
```diff
@@ -0,0 +1,268 @@
...
+You can also run your custom built ClickHouse binary with the config file from the ClickHouse package installed on your system:
+```
+sudo service clickhouse-server stop
+sudo -u clickhouse ClickHouse/build/dbms/programs/clickhouse server --config-file /etc/clickhouse-server/config.xml
+```
+
+
+# IDE (Integrated Development Environment)
+
+If you do not know which IDE to use, we recommend that you use CLion. CLion is a commercial software, but it offers 30 day free trial period. It is also free of charge for students. CLion can be used both on Linux and on Mac OS X.
```
Comment (GoToCoding) [inline dbms/tests/instructions/developer_instruction_en.md]: `we recommend that you use CLion` -> `we recommend you to use CLion`

File: dbms/tests/instructions/developer_instruction_en.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
...
+The Code Style Guide: https://clickhouse.yandex/docs/en/development/style/
+
+Writing tests: https://clickhouse.yandex/docs/en/development/tests/
+
+List of tasks: https://github.com/yandex/ClickHouse/blob/master/dbms/tests/instructions/easy_tasks_sorted_en.md
+
+
+# Test Data
+
+Developing ClickHouse often requires loading realistic datasets. It is particularly important for performance testing. Especially for you we have prepared a set of anonymized data from Yandex.Metrica. It requires additionally some 3GB of free disk space. Note that this data is not required to accomplish most of development tasks.
```
Comment (GoToCoding) [inline dbms/tests/instructions/developer_instruction_en.md]: I think `Especially for you we ...` is better to change just to `We ...`

**Review by GoToCoding — COMMENTED** [review]
LGTM

File: dbms/tests/instructions/developer_instruction_en.md (hunk 5 above)
Comment (Akazz) [reply dbms/tests/instructions/developer_instruction_en.md]: Good point. It doesn't flow well for me neither.

File: dbms/tests/instructions/developer_instruction_en.md (hunk 1 above)
Comment (Akazz) [reply dbms/tests/instructions/developer_instruction_en.md]: A matter of style once again and both would be correct. I take it that the original line meant "if you wish to start working ...". English is very flexible about how you indicate time. I choose to use "will" unless something else suggests it's a conditional or future need.

**Pull request merged**